### ISSUE TITLE
About reporting the training loss when using gradient accumulation

### ISSUE THREAD

**Issue by duynguyen-0203** [espnet/espnet#5090]
**Describe the bug**
During the model training phase, I found a problem when using gradient accumulation. The loss value of the training dataset is divided by the number of accumulation steps before it is reported. This makes it difficult to compare the loss value between the training set and the validation set.
Can you consider changing the position of the two lines of code that I extracted below? Thank you.

https://github.com/espnet/espnet/blob/a505a23332debd897407da5477b3095730e1e512/espnet2/train/trainer.py#L619-L621

**Comment by kamo-naoyuki** [comment]
I got it. This is reasonable. 

@sw005320 Do you mind if I change it?

**Comment by sw005320** [comment]
Yes, this sounds good.
